### ISSUE TITLE
Insert pages from external PDF into PDF in reader

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#3564]
https://forums.zotero.org/discussion/110524/feature-request-add-delete-page-s-from-a-pdf

Easiest would be to insert the whole external PDF at a given point, but we can see what other readers do. (E.g., do they give the option to give a page range to insert?)

In the same thread, someone also asks to insert blank pages.

**Comment by mrtcode** [comment]
Right, therefore for now it could just insert the whole PDF. The unnecessary pages can be deleted manually. And if there will be a need in future to specify a range, we could show a popup. Although all this is probably too much edge case to invest that much work into this.

**Comment by mrtcode** [comment]
https://forums.zotero.org/discussion/comment/455346/#Comment_455346

**Comment by phoeinx** [comment]
Hi everyone,

I came here from searching the Zotero Forums for feature requests for inserting a blank page into a PDF. For me that’s the last feature missing to be able to complete all my annotations/notes in Zotero, which would be awesome!

I’ve seen multiple discussion in the Forum [1,](https://forums.zotero.org/discussion/103862/any-method-to-add-the-blank-page-to-pdf-file-on-zotero) [2,](https://forums.zotero.org/profile/2433/adamsmith), [3](https://forums.zotero.org/discussion/110524/feature-request-add-delete-page-s-from-a-pdf) and the [PR](https://github.com/zotero/pdf-worker/pull/25) opened in May, which lead here. I would love to try to contribute a patch for this feature (which also addresses the open issues raised on the PR from May), but I am unsure on how to proceed best:

Should I 
- open an other discussion on the Zotero Forum for this or 
- work on a patch addressing the points from the last PR, submit it for review as a new PR and see how it goes from there or 
- is that something not worth attempting at the moment, because you are already working on it or it would clash with another feature being implemented or it’s not on the current timeline or something like that?

Either way thanks everyone for your effort for this really great software, I use and enjoy it daily!
